wsmaster/che-core-api-factory-bitbucket-server/src/test/java/org/eclipse/che/api/factory/server/bitbucket/HttpBitbucketServerApiClientTest.java

```diff
@@ -331,7 +331,7 @@ public void shouldBeAbleToGetExistedPAT()
     BitbucketPersonalAccessToken result = bitbucketServer.getPersonalAccessToken("5", "token");
     // then
     assertNotNull(result);
-    assertEquals(result.getToken(), "MTU4OTEwNTMyOTA5Ohc88HcY8k7gWOzl2mP5TtdtY5Qs");
```

Comment:
cc: @artaleks9 isn't it a dummy token?

Reply:
fyi: I created that PR due to a secret scanning alert. If the token is a dummy one, this is not necessary, though I suggest to use dummy tokens that are clearly identifiable as such to avoid false alarms.